Server/Ebenezer/AISocket.cpp

```diff
@@ -155,20 +154,22 @@ void CAISocket::LoginProcess(char* pBuf)
 {
 	int index = 0;
 	float fReConnectEndTime = 0.0f;
-	BYTE ver = GetByte(pBuf, index);
-	BYTE byReConnect = GetByte(pBuf, index);	// 0 : 처음접속, 1 : 재접속
-	CString logstr;
-
+	BYTE zone = GetByte(pBuf, index);
+	// 0: first connect
+	// 1: reconnect
+	BYTE byReConnect = GetByte(pBuf, index);
+	
 	// zone 틀리면 에러 
-	if (ver == -1)
+	if (zone == -1)
```

Comment:
I just want to make this clear that while you didn't do this, this doesn't actually work. It's also bugged in the official 1.298 code. They fixed it in later builds, but this comparison doesn't work.
It needs to be 0xff or 255, or for zone itself to be signed (e.g. int8_t).